README.md

```diff
@@ -1,70 +1,68 @@
-![TiDB Community Logo](media/community-logo.svg)
 
-- [**Discord Channel**](https://discord.gg/KVRZBR2DrG?utm_source=github)
-- [**Slack Channel**](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
-- [**Developer Discussion Forum**](https://internals.tidb.io)
+![](https://pingcap.feishu.cn/space/api/box/stream/download/asynccode/?code=MDM4OTlhYzkyM2ZmYWZiNTkxNGY5M2Y5YjBlNWNlODVfQUZCMUpoUDZoZUFUMGljZWpQWkFtbDUzYkJzeWVnS3FfVG9rZW46TVhIbmJMYTlSb1cyTkh4anlwNWNjcEI4bnZmXzE3MzE0MDEyNTQ6MTczMTQwNDg1NF9WNA)
```

Comment:
recommend to put the assets in the repo (likes media folder) instead of feishu.cn